addons/mail/static/src/new/core/thread_service.js

```diff
@@ -696,6 +696,9 @@ export class ThreadService {
      * @param {Thread} thread
      */
     lastSeenBySelfMessageId(thread) {
+        if (thread.type === "chatter") {
```

Comment:
Should be condition on model being channel instead (and inbox should be ignored too).

Reply:
changes done